runtime/src/rpc.rs

```diff
@@ -133,18 +137,31 @@ impl PolkaBtcProvider {
         ret
     }
 
-    pub async fn get_latest_block_hash(&self) -> Result<Option<H256>, Error> {
-        Ok(self.ext_client.block_hash(None).await?)
+    /// Check that the connection with the parachain works
+    pub async fn check_connection(&self) -> Result<(), Error> {
```

Comment:
I get the intention behind the name change, but this also serves as a utility function. Please revert